docs/proposals/20220215-obsctl.md

```diff
@@ -0,0 +1,67 @@
+# obsctl: A CLI to interact with Observatorium
+
+* **Owners:**
+  * `@onprem` `@saswatamcode`
+
+* **Related Tickets:**
+  * https://issues.redhat.com/browse/MON-2199
+
+* **Other docs:**
+  * None
+
+## Why
+
+We can interact with [Observatorium API](https://github.com/observatorium/api) with a convenient and dedicated CLI, which provides capabilities to persist auth data, and allows us to easily & quickly get up and running with our operations.
+
+## Pitfalls of Current Solution
+
+Currently, we need to manually run [token-refresher](https://github.com/observatorium/token-refresher) with [OIDC](https://openid.net/connect/) information for a particular tenant, and then make cURL requests through that. This is not optimal and generally confusing/trickier for users, both old and new.
+
+## Goals
+
+obsctl will,
+* Manage authentication configuration for multiple tenants by saving them locally
+* Allow users to switch between tenants conveniently
+* Configure and view rules for a tenant
+* View series, rules, and labels for a tenant
+* Later on, support more such one-off operations for other domains as well (logs, traces, alert-routing configs)
+
+## Non-Goals
+
+* Adding and dynamically configuring tenants through obsctl (No such admin API is available within Observatorium, nor is it planned)
```

Comment:
Yup, we can. I was wondering if this would be a future feature though. Maybe leaving it as `nor is it planned in the near future` would be better here?

Reply:
In general setting it as a non-goal here doesn't stop us from revisiting when we have figured out how admin permissions would work in multi-tenant deployments.